app/models/editionable_worldwide_organisation.rb

```diff
@@ -8,6 +8,9 @@ class EditionableWorldwideOrganisation < Edition
   include Edition::Roles
   include Edition::WorldLocations
 
+  include AnalyticsIdentifierPopulator
```

Comment:
I'm assuming we are going to copy these over from the existing orgs, which will mean they are out of sync with the model ID. Shouldn't be a problem, but thought it was worth mentioning.

Reply:
Yes, we will copy them.  My (hopefully not too dangerous) assumption is that the existing Worldwide Organisation IDs are considerably lower than the next Edition ID, so there will never be any duplication of numbering.